pkgs/development/python-modules/python3-xcaplib/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, fetchFromGitHub, buildPythonPackage, isPy3k, python3-application, lxml
+, gevent, python3-eventlib, ... }:
+
+buildPythonPackage rec {
+  pname = "python3-xcaplib";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "AGProjects";
+    repo = "python3-xcaplib";
+    rev = "aadeed49be31ef25cc7a4c32aa7f50fee0e79d72";
```

Comment:
```suggestion
    rev = version;
```

Reply:
There is a tag 2.0.0 (current version), but it is missing some more recent commits that seem to have bug fixes. In my message to the developer's mailing list, I suggested that the version be incremented (and tagged) to account for those changes.

Reply:
I tagged xcaplib

https://github.com/AGProjects/python3-xcaplib/releases/tag/2.0.1

Reply:
Thanks, @adigeo. I've changed to refer to 2.0.1.